ui/components/src/TimeSeriesTooltip/focused-series.ts

```diff
@@ -62,13 +62,12 @@ export function getNearbySeries(
               if (yValue !== '-' && focusedY <= yValue + yBuffer && focusedY >= yValue - yBuffer) {
                 // determine whether to convert timestamp to ms, see: https://stackoverflow.com/a/23982005/17575201
                 const xValueMilliSeconds = xValue > 99999999999 ? xValue : xValue * 1000;
-                const formattedDate = TOOLTIP_DATE_FORMAT.format(xValueMilliSeconds);
                 const formattedY = formatValue(yValue, unit);
                 currentFocusedData.push({
                   seriesIdx: seriesIdx,
                   datumIdx: datumIdx,
                   seriesName: currentSeriesName,
-                  date: formattedDate,
+                  date: xValueMilliSeconds,
```

Comment:
I actually had a PR up for this one in #1138 😂 it looks like the diff's are pretty similar! 

Going to merge your version since I liked that you renamed the param above to `timeMs` 🙌

Reply:
Oh, sorry I didn't see your PR. I did look for open issues containing "tooltip", but I forgot to also look for open PRs 😬

Yeah, they're almost the same - in my first version I used a `Date` object, but later changed it to a number because it's easier to compare in unit tests and there is less struggle with timezones.